### PR TITLE
fix: correctly indent `containerSecurityContext`

### LINKED ISSUE #199: Setting `containerSecurityContext` generates invalid yaml
Setting 

```yaml
zeebe:
  containerSecurityContext:
    capabilities:
      add: ["NET_ADMIN"]
```

generates this yaml: 

```yaml
apiVersion: apps/v1
kind: StatefulSet
metadata:
  name: "default-zeebe"
  labels: ...
  annotations:
spec:
  replicas: 3
  selector:
    matchLabels: ...
  serviceName: "default-zeebe"
  updateStrategy:
    type: RollingUpdate
  podManagementPolicy: Parallel
  template:
    metadata:
      labels: ...
      annotations:
    spec:
      initContainers:
      containers:
      - name: zeebe
        image: "camunda/zeebe:SNAPSHOT"
        imagePullPolicy: Always
        env: ...
        ports: ...
        readinessProbe: ...
        resources: ...
        volumeMounts: ...
        securityContext: capabilities:
              add:
              - NET_ADMIN
```

where the `securityContext` is missing a linebreak.

### PR DESCRIPTION
**Pull request by oleschoenburg**
This fixes #199. I also added a test to verify that the output is correct now.

### CODE REVIEW

File: charts/ccsm-helm/test/zeebe/securitycontext_test.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	t.Parallel()
+
+	chartPath, err := filepath.Abs("../../")
+	require.NoError(t, err)
+
+	suite.Run(t, &golden.TemplateGoldenTest{
+		ChartPath:      chartPath,
+		Release:        "ccsm-helm-test",
+		Namespace:      "ccsm-helm-" + strings.ToLower(random.UniqueId()),
+		GoldenFileName: "containersecuritycontext",
```
Comment (Zelldon) [inline charts/ccsm-helm/test/zeebe/securitycontext_test.go:24]: please add a as prefix the template name (statefulset-)

File: charts/ccsm-helm/test/zeebe/securitycontext_test.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+		Templates:      []string{"charts/zeebe/templates/statefulset.yaml"},
+		SetValues: map[string]string{
+			"zeebe.containerSecurityContext.privileged":          "true",
+			"zeebe.containerSecurityContext.capabilities.add[0]": "NET_ADMIN",
+		},
```
Comment (Zelldon) [inline charts/ccsm-helm/test/zeebe/securitycontext_test.go:29]: :rocket:

File: charts/ccsm-helm/test/zeebe/securitycontext_test.go [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestGoldenContainerSecurityContext(t *testing.T) {
```
Comment (Zelldon) [inline charts/ccsm-helm/test/zeebe/securitycontext_test.go:14]: :x: please move this test as written above :)

**Review by Zelldon — CHANGES_REQUESTED** [review]
Hey @oleschoenburg ! Sorry for any inconvenience I caused and thanks so much for your PR :bow: 

Could you move the test into the statefulset_test.go please? See https://github.com/camunda-community-hub/camunda-cloud-helm/commit/44c785ea682438ae0910f0a129117035b58ab100 I would like to have a test file per template file. 

Really cool that you wrote a test for it :) :medal_military:

**Comment by oleschoenburg** [comment]
@Zelldon Thanks for reviewing :) I rebased to get the new statefulset_test.go file.

**Review by Zelldon — APPROVED** [review]
Thanks Ole :bow:

**Pull request merged**